### ISSUE TITLE
bengali_normalization is not found

### ISSUE THREAD

**Issue by amerzad** [elastic/elasticsearch#27851]
**Elasticsearch version** (`bin/elasticsearch --version`): 6.0.1

**Plugins installed**: []

**JVM version** (`java -version`): Docker image docker.elastic.co/elasticsearch/elasticsearch:6.0.1

**OS version** (`uname -a` if on a Unix-like system): Docker image docker.elastic.co/elasticsearch/elasticsearch:6.0.1

**Description of the problem including expected versus actual behavior**:
Cannot use bengali_normalization filter.

**Steps to reproduce**:
```
PUT doc_bn
{
  "settings": {
    "number_of_shards": 1,
    "number_of_replicas": 0,
    "refresh_interval": "10s",
    "analysis": {
      "filter": {
        "my_stop": {
          "type": "stop",
          "stopwords": "_bengali_"
        },
        "my_stemmer": {
          "type": "stemmer",
          "language": "bengali"
        }
      },
      "analyzer": {
        "my_html": {
          "tokenizer": "standard",
          "char_filter": [
            "html_strip"
          ],
          "filter": [
            "lowercase",
            "indic_normalization",
            "bengali_normalization",
            "my_stop",
            "my_stemmer"
          ]
        }
    }
  },
  "mappings": {
    ...
  }
}

```
The result is:
Custom Analyzer [my_html] failed to find filter under name [bengali_normalization]
It should be available as mentioned in this link: https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-lang-analyzer.html#bengali-analyzer

https://discuss.elastic.co/t/bengali-normalization-not-found/112113/3

**Comment by colings86** [comment]
The bengali analyzer is available from 6.1.0 [1] and is not available in 6.0.1. you will need to upgrade to 6.1.0 to access it.

[1] https://github.com/elastic/elasticsearch/pull/26527

**Comment by amerzad** [comment]
Thanks very much, it was a silly mistake of me!
